kolibri/core/content/utils/import_export_content.py

```diff
@@ -139,7 +139,7 @@ def get_import_export_data(  # noqa: C901
 
     queried_file_objects = {}
 
-    content_ids = set()
+    content_ids = list()
```

Comment:
The only purpose of `content_ids` here is to deduplicate the number of resources - so we can remove most of the logic related to `content_ids` and just keep a count of total number of resources instead.

Reply:
This should still be addressed - instead of keeping a list of `content_ids` we can just start a counter:
```
number_of_resources = 0
```
and increment it by the number of resources within each iteration.